### PR TITLE
Point to solutions GE instance.

### PR DESCRIPTION
**Pull request by runningcode**

### CODE REVIEW

File: .mvn/gradle-enterprise-custom-user-data.groovy [hunk 1]
```diff
@@ -12,4 +12,4 @@ buildScan.tag(project.name)
 buildScan.value('parallel', session.parallel as String)
 
 // Enable storing in the remote build cache based on the presence of a `CI` environment variable
-buildCache.remote.storeEnabled = session.systemProperties.containsKey('CI')
+buildCache.remote.storeEnabled = System.getenv('GITHUB_ACTIONS') != null
```
Comment (etiennestuder) [inline .mvn/gradle-enterprise-custom-user-data.groovy:15]: The comment does not match anymore.

Given this script file is only a sample to show how to use the API, and that we do not really want to be applied when running the build here, I suggest we set another, less impactful (neutral) property on the buildCache API.

**Review by etiennestuder — COMMENTED** [review]
LGTM. Only one suggestion added.

File: .mvn/gradle-enterprise-custom-user-data.groovy (hunk 1 above)
Comment (runningcode) [reply .mvn/gradle-enterprise-custom-user-data.groovy]: I changed it to set storing in the local build cache. How does it look?

Comment (etiennestuder) [reply .mvn/gradle-enterprise-custom-user-data.groovy]: Good idea. LGTM.

File: .mvn/gradle-enterprise-custom-user-data.groovy [hunk 2]
```diff
@@ -11,5 +11,5 @@ buildScan.tag(project.name)
 // Add a custom value based on a property of the `Session`
 buildScan.value('parallel', session.parallel as String)
 
-// Enable storing in the remote build cache based on the presence of a `CI` environment variable
-buildCache.remote.storeEnabled = session.systemProperties.containsKey('CI')
+// Enable storing in the local build cache
```
Comment (etiennestuder) [inline .mvn/gradle-enterprise-custom-user-data.groovy:14]: change comment to:
`// Enable local build caching`

**Review by etiennestuder — APPROVED** [review]
One request for the comment text. can we merged once addressed, without further review.

File: .mvn/gradle-enterprise-custom-user-data.groovy (hunk 2 above)
Comment (runningcode) [reply .mvn/gradle-enterprise-custom-user-data.groovy]: done

**Pull request merged**